weave/refs.py

```diff
@@ -0,0 +1,28 @@
+"""Support for a collection of refs."""
+
+from typing import Iterable, Optional
+
+from rich.table import Table
+
+from weave.trace.refs import parse_uri, AnyRef, CallRef
+
+from weave.trace.rich_container import AbstractRichContainer
+
+
+class Refs(AbstractRichContainer[str]):
+    """A collection of ref strings with utilities."""
+
+    def __init__(self, refs: Optional[Iterable[str]] = None) -> None:
+        super().__init__("Ref", refs)
+
+    def _add_table_columns(self, table: Table) -> None:
+        table.add_column("Ref", overflow="fold")
+
+    def _item_to_row(self, item: str) -> list:
+        return [item]
+
+    def parsed(self) -> list[AnyRef]:
+        return [parse_uri(ref) for ref in self.items]
+
+    def call_refs(self) -> "Refs":
```

Comment:
I'd like to add a `calls` method to this to go directly to a `list[TraceObject]`. This would allow you to write something like:
```
client = weave.init("entity/project")
need_review = client.feedback(reaction="⚠️")
need_review_calls = need_review.refs().calls()
# Save the calls to a dataset or ...
```
But I think this should wait until weave.init settles down.